cypress/features/regression/sidebarWithButton.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Sidebar With Button component
```

Comment:
Could we add event tests?

Reply:
I added this inside of `Sidebar` feature